modules/ding_campaign_plus/ding_campaign_plus.module

```diff
@@ -0,0 +1,395 @@
+<?php
+
+/**
+ * @file
+ * Code for the Ding Campaign Plus feature.
+ */
+
+include_once 'ding_campaign_plus.features.inc';
+
+/**
+ * Implements hook_menu().
+ */
+function ding_campaign_plus_menu() {
+  $items = array();
+
+  $items['admin/config/ding/campaigns_plus'] = array(
+    'title' => 'Campaign plus',
+    'description' => 'Configure campaign plus.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('ding_campaign_plus_admin_settings'),
+    'access arguments' => array('administer site configuration'),
+    'file' => 'includes/ding_campaign_plus.admin.inc',
+  );
+
+  return $items;
+}
+
+/**
+ * Implements hook_permission().
+ */
+function ding_campaign_plus_permission() {
+  return array(
+    'access campaign plus triggers' => array(
```

Comment:
Are this permission used anywhere?

Reply:
Yes it's used in menu callback in the basic triggers. But I think i'll remove it.